### ISSUE TITLE
Experimental Design [Search Result Card]

### ISSUE THREAD

**Issue by jcklpe** [department-of-veterans-affairs/vets-design-system-documentation#1085]
## What

When viewing the facilities under a VAMC the following is returned:

![](https://user-images.githubusercontent.com/9451726/182917862-8429abee-e4b9-4abe-8ee3-05368cbb8a27.png)

Those individual items returned seem ripe for standardization. 

## Purpose

Provides a standardized way to view complex interrelated search item information before clicking through to a detail view. 

## Usage

Currently I only know of it on the Facilities search page for VAMCs

## Behavior

Pretty self contained, I can't think of anything. 

## Examples

see above

## Accessibility

none 

## Guidance

It might be worth expanding this component definition to different kinds of search result items. 

## Next steps

You may present your work to the Design System Council at an upcoming meeting. If you do not or cannot attend the Design Council Meeting, you can opt to get an asynchronous approval.

Submit requests to join an upcoming Design System Council meeting in #platform-design-system.

During the meeting, the Design System Council Working Group will evaluate the request and make a decision.

If your request is approved, you can [add your component or pattern to the system](https://design.va.gov/about/contributing-to-the-design-system/add-a-component-or-pattern-once-approved). If you have any questions on how to add your component or pattern to the system, please reach out to the Design System Team at #platform-design-system.

**Comment by jcklpe** [comment]
A variation of this pattern on also appears on the VA Locations search page:  
  
https://www.va.gov/find-locations/?address=pittsburgh&bounds%5B%5D%5B%5D=-80.740086&bounds%5B%5D%5B%5D=39.691694&bounds%5B%5D%5B%5D=-79.240086&bounds%5B%5D%5B%5D=41.191694&bounds%5B%5D%5B%5D%5B%5D=-80.740086&bounds%5B%5D%5B%5D%5B%5D=39.691694&bounds%5B%5D%5B%5D%5B%5D=-79.240086&bounds%5B%5D%5B%5D%5B%5D=41.191694&context=Pittsburgh%2C%20Pennsylvania%2C%20United%20States&facilityType=vet_center&latitude=40.441694&longitude=-79.990086&page=1&radius=28&serviceType&bounds%5B%5D=-80.740086&bounds%5B%5D=39.691694&bounds%5B%5D=-79.240086&bounds%5B%5D=41.191694  
  
  
![](https://user-images.githubusercontent.com/9451726/182930764-7739e08c-5114-46af-b182-6c87da1fa830.png)

**Comment by jcklpe** [comment]
Also apparently some of the previous work was done here: https://www.sketch.com/s/433403fb-b16c-4c98-867e-181d86208ba2

**Comment by caw310** [comment]
This is on the agenda for DSC meeting at 1pm ET on 8/5.

**Comment by caw310** [comment]
Follow up from DSC meeting: 
A few weeks ago DSC we talked about creating a shared library. Search was one of those things that could be a shared library. It would be appropriate to start a search shared library and put location in it. 

Action items:
Create template for Search shared library in Search (Matt, Natalie & Sean)
Set up time with Aslan to talk through how they create things in Sketch. (Matt, Natalie & Sean)
Document shared libraries - (Matt)

**Comment by humancompanion-usds** [comment]
We have created the first Shared Library which is for Search: https://www.sketch.com/s/ab0f611b-c15c-42c4-ab71-1158ff6e01c6

Megan has agreed to be our maintainer, for now. We've also documented Shared Libraries here:
https://design.va.gov/about/for-designers/design-libraries#shared-libraries

We'll now announce the shared library and the documentation in the Monday meeting and via Slack. After that we can close this issue.
